tools/snp.sh

```diff
@@ -465,6 +465,28 @@ set_grub_default_snp() {
   sudo update-grub
 }
 
+grubby_to_set_grub_default_snp(){
```

Comment:
grubby_set...

Reply:
Removed grubby from this script

Modified RHEL grub configuration and reloaded settings in "rhel_set_grub_default_snp"